src/Framework/Interaction/Target.cxx

```diff
@@ -80,7 +83,8 @@ fA(0),
 fTgtPDG(0),
 fHitNucPDG(0),
 fHitSeaQrk(false),
-fHitNucP4(0)
+fHitNucP4(0),
+fHitEleP4(0)
```

Comment:
Do we really need this? Meaning. You cannot hit at the same time both the nucleon and the electron. So you can store the electron momentum information in the nucleon. Did you think about it? What troubles do we get ourselves into if we avoid creating more memory?

Reply:
I think we'll talk about this